spark/src/main/scala/ai/chronon/spark/SparkSessionBuilder.scala

```diff
@@ -90,7 +91,8 @@ object SparkSessionBuilder {
     }
     val spark = builder.getOrCreate()
     // disable log spam
-    spark.sparkContext.setLogLevel("ERROR")
+    // spark.sparkContext.setLogLevel("ERROR")
```

Comment:
do you want to keep this line?

Reply:
I think there is an issue with logs right now - will follow up PR to fix the logs issue